src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -92,31 +123,100 @@ public override Task SendUsersAsync(IReadOnlyList<string> userIds, string method
 
         public Task UserAddToGroupAsync(string userId, string groupName, CancellationToken cancellationToken = default)
         {
+            if (string.IsNullOrEmpty(userId))
```

Comment:
The validation of userId and groupName happens more than once. Could you please merge them to a standalone validation method?

public void ValidateUserIdAndGroupName(string userId, string groupName)
{
    xxxxx
}

Then in other place, just call ValidateUserIdAndGroupName(userId, groupName).